lib/bugsnag/notification.rb

```diff
@@ -410,6 +410,17 @@ def stacktrace(exception)
       end.compact
     end
 
+    def in_project?(line)
+      return nil if @configuration.vendor_paths && @configuration.vendor_paths.any? do |vendor_path|
```

Comment:
I think we should return false here rather than nil to honour the ? convention in ruby.